tozny/resource_realm_role.go

```diff
@@ -63,6 +63,31 @@ func resourceRealmRole() *schema.Resource {
 				Computed:    true,
 				ForceNew:    true,
 			},
+			"attribute": {
+				Description: "Arbitrary string-string key value pairs.",
+				Type:        schema.TypeList,
+				Optional:    true,
+				ForceNew:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"key": {
+							Description: "The key to use for the attribute",
+							Type:        schema.TypeString,
+							Required:    true,
+							ForceNew:    true,
```

Comment:
do u need to recreate or can u just update?

Reply:
If the attributes exist then we would just need to update or else create them but these align with how the other attributes are defined as well so basically any change to role will eventually recreate the role

Reply:
if it matches how attributes are in groups and other files its okay

Reply:
yeah - I just used the same one from groups